lib/ansible/utils/module_docs_fragments/ironware.py

```diff
@@ -0,0 +1,92 @@
+#
+# (c) 2016, Peter Sprygada <psprygada@ansible.com>
+# (c) 2016, Patrick Ogenstad <@ogenstad>
```

Comment:
Please remove above two line and add your name as maintainer. Also, please update the year (applicable for headers in other files as well.)